easybuild/easyconfigs/v/vip-report/vip-report-v0.0.8-Java-8-LTS.eb

```diff
@@ -0,0 +1,37 @@
+name = 'vip-report'
```

Comment:
vip-report required Java 11 and doesn't function correctly with Java 8. This file should be removed.